src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -118,6 +120,10 @@ const ActiveState = ({
     onVersionInfo,
     translations,
 }: Props): React.Node => {
+    const onThreadSelected = (parentId: string) => (isSelected: boolean) => {
```

Comment:
Should this be something like `const onItemFocusHandler = (itemId: string)`? It's not specific to threads or replies, since it's used for `Comment` below.